README.md

```diff
@@ -74,7 +74,7 @@ If you want to build your own app using Q-municate as a base, please do the foll
 3. Run `npm install -g bower` in your terminal as an administrator.
 4. Run `gem install compass` in your terminal as an administrator.
 5. Run `npm install -g grunt-cli` in your terminal as an administrator.
-6. Run `bower install` to install all additional packages in your terminal.
```

Comment:
We don't use any devDependencies in bower.json  (https://github.com/QuickBlox/q-municate-web/blob/master/bower.json).